### DISCUSSION TITLE
Need extra help and something answered

### DISCUSSION THREAD

**Discussion by jhjhfjdhfjdhfjd** [vercel/community#3751] [Account Issue]
### Question

Basically, I emailed them and told them I was gonna make python scripts for Discord in the email so I can get my account unblocked but they haven't responded in 2 days! help.

(Backstory is like my account was blocked at registration and I had to email them and they didnt respond after my first reply to them....)

**Comment by amyegan** [comment]
Due to the sensitive nature of account info, we cannot safely discuss it in this public forum. Duplicate of https://github.com/vercel/community/discussions/3710
